backend/private-graph/graph/schema.resolvers.go

```diff
@@ -7864,6 +7864,24 @@ func (r *sessionResolver) DeviceMemory(ctx context.Context, obj *model.Session)
 	return deviceMemory, nil
 }
 
+// SessionComments is the resolver for the session_comments field.
+func (r *sessionResolver) SessionComments(ctx context.Context, obj *model.Session) ([]*model.SessionComment, error) {
+	if util.IsDevEnv() && obj.SecureID == "repro" {
+		sessionComments := []*model.SessionComment{}
+		return sessionComments, nil
+	}
+	s, err := r.canAdminViewSession(ctx, obj.SecureID)
+	if err != nil {
+		return nil, err
+	}
+	sessionComments := []*model.SessionComment{}
+
+	if err := r.DB.Preload("Attachments").Preload("Replies").Where(model.SessionComment{SessionId: s.ID, Type: model.SessionCommentTypes.FEEDBACK}).Order("timestamp asc").Find(&sessionComments).Error; err != nil {
```

Comment:
Query session comments with
```golang
Type: model.SessionCommentTypes.FEEDBACK
```

Reply:
could you rename the field from `session_comments` to `session_feedback` so it's clear that only feedback is included in the results? I also think you can omit `.Preload("Attachments").Preload("Replies")` since I don't think those fields are used for feedback?

Reply:
I've made the relevant changes.